examples/groups/README.md

```diff
@@ -29,23 +79,8 @@ No provider.
 
 | Name | Description | Type | Default | Required |
 |------|-------------|------|---------|:--------:|
-| account\_alias | Assign the account alias for the AWS Account. Unmanaged by default. Resource will be created if the string is non-empty. | `string` | `""` | no |
-| account\_pass\_policy | Manages Password Policy for the AWS Account. Unmanaged by default. Resource will be created if 'manage' is set to true. | <pre>object({<br>    manage                         = bool   # Set to true, to manage the AWS account password policy<br>    allow_users_to_change_password = bool   # Allow users to change their own password?<br>    hard_expiry                    = bool   # Users are prevented from setting a new password after their password has expired?<br>    max_password_age               = number # Number of days that an user password is valid<br>    minimum_password_length        = number # Minimum length to require for user passwords<br>    password_reuse_prevention      = number # The number of previous passwords that users are prevented from reusing<br>    require_lowercase_characters   = bool   # Require lowercase characters for user passwords?<br>    require_numbers                = bool   # Require numbers for user passwords?<br>    require_symbols                = bool   # Require symbols for user passwords?<br>    require_uppercase_characters   = bool   # Require uppercase characters for user passwords?<br>  })</pre> | <pre>{<br>  "allow_users_to_change_password": null,<br>  "hard_expiry": null,<br>  "manage": false,<br>  "max_password_age": null,<br>  "minimum_password_length": null,<br>  "password_reuse_prevention": null,<br>  "require_lowercase_characters": null,<br>  "require_numbers": null,<br>  "require_symbols": null,<br>  "require_uppercase_characters": null<br>}</pre> | no |
-| providers\_saml | A list of dictionaries defining saml providers. | <pre>list(object({<br>    name = string # The name of the provider to create<br>    file = string # Path to XML generated by identity provider that supports SAML 2.0<br>  }))</pre> | `[]` | no |
-| providers\_oidc | A list of dictionaries defining openid connect providers. | <pre>list(object({<br>    url             = string       # URL of the identity provider. Corresponds to the iss claim<br>    client_id_list  = list(string) # List of client IDs (also known as audiences)<br>    thumbprint_list = list(string) # List of server certificate thumbprints.<br>  }))</pre> | `[]` | no |
 | policies | A list of dictionaries defining all policies. | <pre>list(object({<br>    name = string      # Name of the policy<br>    path = string      # Defaults to 'var.policy_path' variable is set to null<br>    desc = string      # Defaults to 'var.policy_desc' variable is set to null<br>    file = string      # Path to json or json.tmpl file of policy<br>    vars = map(string) # Policy template variables {key: val, ...}<br>  }))</pre> | `[]` | no |
```

Comment:
We should try to stay consistent: maps vs. dictionaries :)

Reply:
@danvaida the only thing which is not a list of dictionaries is `account_pass_policy`, which can only be set once. All others allow for multiple values. Is it that what you meant?

Reply:
What I meant is that we are referring to the same variable type as _map_ but also as _dictionary_.
Technically, both are correct, but for the sake of consistency and to avoid confusion, I suggest we use one or the other, not both.